next/src/components/common/CheckBoxGroup/Checkbox.tsx

```diff
@@ -0,0 +1,48 @@
+import { Checkbox as RACCheckbox, CheckboxProps as RACCheckboxProps } from 'react-aria-components'
+
+import { CheckIcon } from '@/src/assets/icons'
+import cn from '@/src/utils/cn'
+
+/**
+ * Figma: https://www.figma.com/design/17wbd0MDQcMW9NbXl6UPs8/DS--Component-library?node-id=16717-20069&t=nFl2Kvs9pZLewXiT-4
+ * Based on RAC library: https://react-spectrum.adobe.com/react-aria/Checkbox.html
+ */
+
+const Checkbox = ({ children, value, ...restProps }: RACCheckboxProps) => {
+  return (
+    <RACCheckbox
+      key={value}
+      value={value}
+      className={({ isHovered }) => {
+        return cn('base-focus-ring flex items-center gap-3 rounded-sm', {
+          'text-content-active-primary-hover': isHovered,
+        })
+      }}
+      {...restProps}
+    >
+      {({ isSelected, isHovered, isDisabled }) => {
+        return (
+          <>
+            <div
+              className={cn(
+                'flex size-6 shrink-0 items-center justify-center rounded-md border-[0.2rem] border-border-active-primary-default text-content-passive-inverted-primary',
```

Comment:
I think in Tailwind 4 you can just use `border-0.5` without defining it in config. Yes, it's in px, but I guess for borders it's desired (?). But as you prefer, no need for further discussion.